src/api/db/project_db.py

```diff
@@ -1,6 +1,8 @@
 """
 Database operations for projects using individual YAML files.
 """
+from http.client import HTTPException
+from logging import info
```

Comment:
```suggestion
from utils.logging import info
```

The project has its own custom logging class.